knowledge-base/common-change-selected-localization-keys.md

```diff
@@ -0,0 +1,197 @@
+    app.MapFallbackToPage("/_Host");
+
+    app.Run();
+    ````
+3. Add a cookie in the `_Host.cshtml` file to store the culture choice:
+    >caption _Host.cshtml
+    ````CSHTML
+    @using Microsoft.AspNetCore.Localization
+    @using System.Globalization
+
+    . . . .
+    <body class="k-body">
+
+        @* Culture cookie start *@
+
+        @{
+            this.HttpContext.Response.Cookies.Append(CookieRequestCultureProvider.DefaultCookieName,
+                CookieRequestCultureProvider.MakeCookieValue(new RequestCulture(CultureInfo.CurrentCulture, CultureInfo.CurrentUICulture))
+            );
+        }
+
+        @* Culture cookie end *@
+
+        <app>
+            <component type="typeof(App)" render-mode="ServerPrerendered" />
+        </app>
+
+    . . . .
+
+    </body>
+    ````
+4. Create a services folder and implement the Telerik localization service with a list of all needed localization keys and their corresponding custom strings. The service also relies on a `~/Resources` folder with the necessary `.resx` files. You can find an up-to-date list of the used strings in our [API documentation](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Resources.Messages) and the [offline version](https://www.telerik.com/account/my-downloads) of our [Demo solution](https://demos.telerik.com/blazor-ui). Telerik maintains the main `TelerikMessages.resx` file (in English) to have the full list with each release.
+    >caption ResxLocalizer.cs
+    ````CSHTML
+    using Telerik.Blazor.Services;
+
+    public class ResxLocalizer : ITelerikStringLocalizer
+    {
+        private readonly ITelerikStringLocalizer _fallback;
+
+        //here is the list of the desired localization keys with their custom string values
+        private Dictionary<string, string> _ownStrings = new Dictionary<string, string>()
+        {
+            { "Wizard_Next", "Your Custom Text" }
+        };
+
+        public ResxLocalizer()
+        {
+            // this is your public implementation of ITelerikStringLocalizer!
+            _fallback = new TelerikStringLocalizer();
+        }
+
+        public string this[string name]
+        {
+            get
+            {
+                if (_ownStrings?.ContainsKey(name) ?? false)
+                {
+                    return _ownStrings[name];
+                }
+                else
+                {
+                    return _fallback[name];
+                }
+            }
+        }
+    }
+    ````
+4(continued). Add the desired `.resx` files to the `~/Resources` folder.
+
+    Name the files like this `~/Resources/TelerikMessages.<culture-locale>.resx`, for example `TelerikMessages.bg-BG.resx`. You can use different names (for example, in our demos we use `TelerikMessages.resx`). The file names affect the static class that is generated and how you use it in your code (for example, to localize other elements you define yourself, such as grid command buttons or your own buttons).
```

Comment:
Make the three paragraphs below a nested list (nested under step 5 above). They don't seem to follow a specific sequence, so use a bullet list.